mappings/net/minecraft/util/math/Vec3d.mapping

```diff
@@ -1,81 +1,208 @@
 		ARG 1 vec
 	METHOD method_1023 subtract (DDD)Lnet/minecraft/class_243;
+		COMMENT Subtracts the given vector from this vector.
+		COMMENT
+		COMMENT @see #reverseSubtract(Vec3d)
+		COMMENT @return the difference between this vector and the given vector
 		ARG 1 x
 		ARG 3 y
 		ARG 5 z
 	METHOD method_1024 rotateY (F)Lnet/minecraft/class_243;
+		COMMENT Rotates this vector by the given angle around the Y axis.
 		ARG 1 angle
+			COMMENT the angle in radians
 	METHOD method_1025 squaredDistanceTo (Lnet/minecraft/class_243;)D
+		COMMENT Returns the squared distance between this vector and the given vector.
+		COMMENT
+		COMMENT <p>Can be used for fast comparison between distances.
+		COMMENT
+		COMMENT @see #squaredDistanceTo(double, double, double)
+		COMMENT @see #distanceTo(Vec3d)
 		ARG 1 vec
 	METHOD method_1026 dotProduct (Lnet/minecraft/class_243;)D
+		COMMENT Returns the dot product of this vector and the given vector.
 		ARG 1 vec
 	METHOD method_1027 lengthSquared ()D
+		COMMENT Returns the squared length of this vector.
+		COMMENT
+		COMMENT <p>Can be used for fast comparison between lengths.
+		COMMENT
+		COMMENT @see #length()
 	METHOD method_1028 squaredDistanceTo (DDD)D
+		COMMENT Returns the distance between this vector and the given vector.
+		COMMENT
+		COMMENT <p>Can be used for fast comparison between distances.
+		COMMENT
+		COMMENT @see #squaredDistanceTo(Vec3d)
+		COMMENT @see #distanceTo(Vec3d)
 		ARG 1 x
 		ARG 3 y
 		ARG 5 z
 	METHOD method_1029 normalize ()Lnet/minecraft/class_243;
+		COMMENT Normalizes this vector.
+		COMMENT
+		COMMENT <P>Normalized vector is a vector with the same direction but with
+		COMMENT length 1. Each coordinate of normalized vector has value between 0
+		COMMENT and 1.
+		COMMENT
+		COMMENT @return the normalized vector of this vector
 	METHOD method_1030 fromPolar (FF)Lnet/minecraft/class_243;
+		COMMENT Converts pitch and yaw into a direction vector.
+		COMMENT
+		COMMENT @see #fromPolar(Vec2f)
 		ARG 0 pitch
 		ARG 1 yaw
 	METHOD method_1031 add (DDD)Lnet/minecraft/class_243;
+		COMMENT Returns the sum of this vector and the given vector.
+		COMMENT
+		COMMENT @see #add(Vec3d)
 		ARG 1 x
 		ARG 3 y
 		ARG 5 z
 	METHOD method_1032 floorAlongAxes (Ljava/util/EnumSet;)Lnet/minecraft/class_243;
+		COMMENT Applies the floor function to the coordinates chosen by the given axes.
 		ARG 1 axes
 	METHOD method_1033 length ()D
+		COMMENT Returns the length of this vector.
+		COMMENT
+		COMMENT <p>The length of a vector is equivalent to the distance between that
+		COMMENT vector and the {@linkplain #ZERO} vector.
+		COMMENT
+		COMMENT @see #lengthSquared()
 	METHOD method_1034 fromPolar (Lnet/minecraft/class_241;)Lnet/minecraft/class_243;
+		COMMENT Converts pitch and yaw into a direction vector.
+		COMMENT
+		COMMENT @see #fromPolar(float, float)
 		ARG 0 polar
+			COMMENT the vector composed of pitch and yaw
 	METHOD method_1035 reverseSubtract (Lnet/minecraft/class_243;)Lnet/minecraft/class_243;
```

Comment:
This should be `relativize`, though the name is hard to think of when you first look at the code. But if you think of it as using this vector as a vector end and the given vector as a vector start, it clearly is `relativize` (I thought about this because `reverseSubtract` is too unnatural)